src/controllers/buyProduct.controller.js

```diff
@@ -0,0 +1,57 @@
+import Cart from '../models/cart';
+import Product from '../models/Product';
+
+export class buyProduct {
+  static async getPdts(req, res, next) {
+    try {
+      const pdts = await Product.find({});
+      return res.status(200).json({
+        status: 'success',
+        message: 'found products',
+        data: pdts
+      });
+    } catch (error) {
+      return next(error);
+    }
+  }
+
+  static async myCart(req, res, next) {
+    Cart.checkOut = true;
+    await Cart.save();
+    res.json({ products: `${Cart.items} bought` });
+    next();
+    return res.json({ message: "couldn't purchase" });
+  }
+
+  static async addToCart(req, res, next) {
+    try {
+      const pdt = await Product.findById(req.params.id);
```

Comment:
How can I test this, adding products/product to the cart?

Reply:
You pass in a product id, and it'll find it in the db and add it to the cart